client/pages/my-builds.tsx

```diff
@@ -32,7 +32,7 @@ const MyBuildsPage: NextPage = () => {
         />
         <title>{getTitle(t('MY_BUILDS'))}</title>
       </Head>
-      <MyBuilds isMobile />
+      <MyBuilds username={data?.currentUser?.username!} />
```

Comment:
I think you could just do `{data?.currentUser && <MyBuilds username={data.currentUser.username} />}` since this page is meaningless if the user is not logged in and the useEffect should quickly route them somewhere else.